.github/ISSUE_TEMPLATE/release-patch.md

```diff
@@ -0,0 +1,94 @@
+---
+name: NEURON Patch Release
+about: Create a NEURON Patch release for an existing branch.
+title: 'NEURON [x.y.z] patch release'
+labels: 'release'
+assignees: ''
+
+---
+
+Action items
+============
+
+Pre-release
+---
+- [ ] Create a cherrypicks branch where all commits go into new release and open a PR against `realease/x.y` branch
+- [ ] Create CoreNEURON tag on the `release/x.y` branch after cherrypicking required commits, update semantic version in its `CMakeLists.txt`, tag it & update submodule in cherrypicks PR
+- [ ] Look out for ModelDB regressions by manually submitting and analyzing [nrn-modeldb-ci](https://github.com/neuronsimulator/nrn-modeldb-ci/actions/workflows/nrn-modeldb-ci.yaml?query=event%3Aschedule++) for the cherrypicks branch vs previous version
+- [ ] Update cherrypicks PR:
+  - [ ] Update semantic version in `CMakeLists.txt`
+  - [ ] Update changelog below and agree on it with everyone; then commit it to `docs/changelog` in the cherrypicks PR (copy structure as-is)
+  - [ ] Update `docs/index.rst` accordingly with the new `.pkg` and `.exe` links for `PKG installer` and `Windows Installer`
+
+Sanity checks
+---
+- [ ]  After cherrypicks PR is merged, make sure GitHub, Azure and CircleCI builds pass for `release/x.y` branch
+- [ ] Run [nrn-build-ci](https://github.com/neuronsimulator/nrn-build-ci/actions/workflows/build-neuron.yml) for the `release/x.y` branch; see [nrn-build-ci guide](https://github.com/neuronsimulator/nrn-build-ci#azure-wheels-testing---manual-workflow) 
+- [ ] Activate ReadTheDocs build for `release/x.y` & make it hidden. Check docs are fine after build is done.
+- [ ] Run BBP Simulation Stack, CoreNEURON CI & other relevant tests
+- [ ] Build release wheels but WITHOUT upload ([see details](https://nrn.readthedocs.io/en/latest/install/python_wheels.html#publishing-the-wheels-on-pypi-via-azure))
+
+
+Releasing
+---
+- [ ] Create new release+tag on GitHub via [release workflow](https://github.com/neuronsimulator/nrn/actions/workflows/release.yml?query=workflow%3A%22NEURON+Release%22)
+- [ ] Create, test and upload manual artifacts 
+  - [ ] MacOS package installer (manual task, ask Michael)
+  - [ ] arm64 wheels (manual task, check with Alex or Pramod)
+  - [ ] aarch64 wheels (use existing `release/x.y-aarch64` branch for this, see [guide](https://nrn.readthedocs.io/en/latest/install/python_wheels.html#publishing-the-wheels-on-pypi-via-circleci))
+- [ ] Publish the `x.y.z` wheels on Pypi; see [wheel publishing instructions](https://nrn.readthedocs.io/en/latest/install/python_wheels.html#publishing-the-wheels-on-pypi-via-azure)
+- [ ] Once wheels are published, activate the `x.y.z` tag on ReadTheDocs
+- [ ] Upload Windows installer from the wheels publishing Azure run (to get correct tag)
+- [ ] Publish release on GitHub (edit https://github.com/neuronsimulator/nrn/releases/tag/x.y.z)
```

Comment:
Maybe link to https://github.com/neuronsimulator/nrn/releases/new or wherever you think is best? Explicitly say it should be an annotated (not lightweight) tag? (IIRC this is important for `git describe`?)

Reply:
The release workflow takes care of the annotated tag and creating the pre-release on GH. This task is just to publish (untick pre-release)